### DISCUSSION TITLE
How to render with alpha

### DISCUSSION THREAD

**Discussion by Cherryke** [cables-gl/cables_docs#509] [Answers & Questions] [answered]
I am trying to save a clip with the Mediarecorder in order to process it in Blender (and AE). Which codec can I use? Is there a tutorial for this? 
I have set the background in Cables to transparent but its black...

**Comment by steam0r** [comment, marked as answer]
MediaRecorder capabilities depend on what your browser can do. it will give you all the options in the dropdown. 
i think alpha only works with webm.

there is a youtube tutorial on MediaRecorder: https://www.youtube.com/watch?v=zWfwC-GH7qk
